### PR TITLE
Adding "on" validation on $_SERVER['HTTPS') variable.

### PR DESCRIPTION
**Pull request by Payetus**
This commit should fix issue #108

### CODE REVIEW

File: src/helpers.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ function str_rot_pass($str, $key, $decrypt = false){
 }
 
 function app_url(){
-	return (!empty($_SERVER['HTTPS']) ? 'https://' : 'http://').$_SERVER['HTTP_HOST'].$_SERVER['PHP_SELF'];
+	return (!empty($_SERVER['HTTPS'])&& $_SERVER['HTTPS']=="on" ? 'https://' : 'http://').$_SERVER['HTTP_HOST'].$_SERVER['PHP_SELF'];
```
[the remark this answers is not in the document]
Comment (Payetus) [reply src/helpers.php]: Okay

**Comment by Athlon1600** [comment]
according to PHP docs, ``` $_SERVER['HTTPS'] ``` does not necessarily have to be set to "on" for it to be an HTTPS request:  
http://php.net/manual/en/reserved.variables.server.php  
  
From what I understand, as long as server[https] is not empty and not set to "off", then that's an https request. Are you using IIS by any chance?

**Comment by Payetus** [comment]
Yes, using IIS 8. The $_SERVER['HTTPS'] is set to off (and obviously not empty ) on http requests. Would you attribute this to an IIS bug?

**Comment by Athlon1600** [comment]
it's been changed to this now:
image

**Pull request closed without merging**